pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -496,8 +512,13 @@ func (m *Manager) backgroundMetricsFlush(ctx context.Context) {
 			sinceLastRecord := m.metrics.recordConsumption(consumptionInfo, keyspaceName, m.controllerConfig, now)
 			resourceGroupName := consumptionInfo.resourceGroupName
 			// TODO: maybe we need to distinguish background ru.
-			if rg := m.GetMutableResourceGroup(keyspaceID, resourceGroupName); rg != nil {
+			if rg, err := m.GetMutableResourceGroup(keyspaceID, resourceGroupName); rg != nil {
 				rg.UpdateRUConsumption(consumptionInfo.Consumption)
+			} else {
+				log.Error("failed to get mutable resource group",
+					zap.Uint32("keyspace-id", keyspaceID),
+					zap.String("resource-group-name", resourceGroupName),
+					zap.Error(err))
```

Comment:
I feel that this log may not be very necessary. If the corresponding group is deleted during the workload, the reporting frequency might become very frequent due to the load. Perhaps deleting it or changing it to debug would be more appropriate.